### PR TITLE
Fix: typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos

### CODE REVIEW

**Review by seanyoung — APPROVED** [review]

File: docs/targets/solana.rst [hunk 1]
```diff
@@ -191,10 +191,10 @@ or a hex string with the format ``hex"4142"``, or a constructor argument of type
 
 .. _value_transfer:
 
-Transfering native value with a function call
+Transferring native value with a function call
 _____________________________________________
```
Comment (seanyoung) [inline docs/targets/solana.rst:195]: The title is now longer so you need an extra `_` to make the tests pass, thanks

Comment (omahs) [reply docs/targets/solana.rst]: Sure! Just added it

**Comment by seanyoung** [comment]
@omahs one more thing is needed to merge your request: we need a  `Signed-off-by:` with your real name and email address. See https://wiki.hyperledger.org/display/TSC/DCO+and+Pseudonyms

Thanks

**Comment by omahs** [comment]
@seanyoung Interesting, I contributed to some other hyperledger repos and never had to do this, I'd rather not publish my private email here.
Signed-off by: Omahs, 73983677+omahs@users.noreply.github.com

**Comment by seanyoung** [comment]
> @seanyoung Interesting, I contributed to some other hyperledger repos and never had to do this, I'd rather not publish my private email here. Signed-off by: Omahs, [73983677+omahs@users.noreply.github.com](mailto:73983677+omahs@users.noreply.github.com)

Turns out there is no hyperledger-wide rules on this, see discussion in [#maintainers-discussion](https://discord.com/channels/905194001349627914/941386911459737692). It is up to the individual hyperledger projects to set rules.

So we would like a real email address so that we can reach contributors for issues like re-licensing. Also, the `Signed-off-by` is meaningless if a pseudonym is used. So, please do your real email address and your real please - we appreciate it. Thanks.

**Comment by omahs** [comment]
`Signed-off-by:` Omahs Alief, omahsalief@outlook.com

**Review by LucasSte — APPROVED** [review]

**Comment by seanyoung** [comment]
@omahs thank you very much!

**Pull request merged**